core/app/components/FormBuilder/ElementPanel/ComponentConfig/MultivalueBase.tsx

```diff
@@ -0,0 +1,71 @@
+import type { ReactNode } from "react";
+
+import { CoreSchemaType } from "db/public";
+import { FormControl, FormDescription, FormField, FormItem, FormLabel, FormMessage } from "ui/form";
+import { Input } from "ui/input";
+import { MultiValueInput } from "ui/multivalue-input";
+
+import type { InnerFormProps } from "./types";
+
+export default ({
+	form,
+	schemaName,
+	label = "Dropdown",
+	children,
+}: InnerFormProps & { label?: string; children?: ReactNode }) => {
+	const isNumeric = schemaName === CoreSchemaType.NumericArray;
+	return (
+		<div className="flex flex-col gap-6">
+			<FormField
+				control={form.control}
+				name="config.label"
+				render={({ field }) => (
+					<FormItem>
+						<FormLabel>Label</FormLabel>
+						<FormControl>
+							<Input placeholder="Description of the selection" {...field} />
+						</FormControl>
+						<FormMessage />
+					</FormItem>
+				)}
+			/>
+			<FormField
+				control={form.control}
+				name="config.help"
+				render={({ field }) => (
+					<FormItem>
+						<FormLabel>Description</FormLabel>
+						<FormControl>
+							<Input placeholder="Optional additional guidance" {...field} />
+						</FormControl>
+						<FormDescription>Appears below the label</FormDescription>
+						<FormMessage />
+					</FormItem>
+				)}
+			/>
+			<FormField
+				control={form.control}
+				name="config.values"
+				render={({ field }) => (
+					<FormItem>
+						<FormLabel>{label} Values</FormLabel>
+						<FormControl>
+							<MultiValueInput
+								type={isNumeric ? "number" : undefined}
+								onChange={(e: string[]) => {
+									if (isNumeric) {
+										return field.onChange(e.map((v) => +v));
+									}
+									field.onChange(e);
+								}}
+								value={field.value ?? []}
+							/>
```

Comment:
Clicking the drag handle on an added option causes the form panel to close. I'm guessing it's a button without a type? It's not so bad since it's saving but probably worth fixing!

https://github.com/user-attachments/assets/c05d736e-5a90-4ea2-8937-061a1295baae

Reply:
also, given how much this comes up, maybe we should default our Button component to `type="button"`? I doubt we're relying on that defaulting to `submit` anywhere

Reply:
I'd be in favor of setting `type="button"` to be the default, `type="submit"` as the default (in general) was always confusing to me. I get that that was the only way you could use a button in ye olden days, but I think specifically saying "This button is used to submit a form" is obviously the case you would want to call out by setting the `type`.

Reply:
oh great catch! I've confirmed that changing the Button component's `type` to `button` fixes things, so I'll make that the default 👍

Reply:
well i guess we were relying on that in some places. but i'm glad we're not now!